tests/acceptance/reminders/edit-test.js

```diff
@@ -0,0 +1,39 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'remember/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | reminders/edit');
+
+test('editing a reminder title changes its title in list', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('/reminders/edit/1');
+
+  andThen(function() {
+    fillIn('.reminder-title', 'Hello World');
+    click('.btn-update');
+
```

Comment:
Ok we can try to change the route...it will probably take significant refactoring